nisext/sexts.py

```diff
@@ -2,10 +2,10 @@
 
 import os
 from configparser import ConfigParser
-from distutils import log
-from distutils.command.build_py import build_py
-from distutils.command.install_scripts import install_scripts
-from distutils.version import LooseVersion
+import logging as log
+from setuptools.command.build_py import build_py
+from setuptools.command.install_scripts import install_scripts
+from looseversion import LooseVersion
```

Comment:
On the off-chance that anybody's using `nisext`, it would be a pain to cause them to fail this import because `looseversion` is not a dependency.
```suggestion
from setuptools._distutils.version import LooseVersion
```

Reply:
I'm okay with using `setuptools._distutils.version` instead of `looseversion`. But I also put `looseversion` as a dependency in `pyproject.toml`. Unless, I'm missing something, that should pull in `loosversion` when `nibabel` is installed.

For me, at least, both approaches work.

Reply:
> For me, at least, both approaches work.

I was wrong. The dependency is not resolved automatically in the build environment. It was still cached from an earlier build. I dropped `loosversion` in the latest commit.

PS: I meant to comment and resolve at the same time. But that didn't work.